### ISSUE TITLE
Change assets and uploaded_documents defaults to separate directories

### ISSUE THREAD

**Issue by RafaPolit** [huridocs/uwazidocs#4511]
Due to logistics and some security concerns to better allow for permissions filtering and static file serving, it would be ideal if "assets" path and "uploaded_documents" are not the same physical directory.  Our default for those is the same physical directory.  Although it can be changed via config and we should encourage that, we should also change the defaults.

**Comment by txau** [comment]
By the assets you mean the custom uploads?

**Comment by RafaPolit** [comment]
Yes, that is the name of the route that fetches what we call in the UI "custom uploads".  We should also make an effort to unify the names.

**Comment by aphilop** [comment]
Resolved with Files V2
